src/app/components/mnoe-users-local-list/mnoe-users-local-list.html

```diff
@@ -40,8 +40,13 @@
             <span ng-show="user.status === 'invited'" ng-click="removeUserFromOrganization(user)" uib-tooltip="{{'mnoe_admin_panel.dashboard.users.widget.local_list.cancel_invite' | translate}}"><i class="fa fa-close fa-1"></i></span>
           </td>
           <td ng-if="organizationContext">
-            <select ng-model="user.role" ng-options="role as keyFromRole(role) | translate for role in availableRoles" ng-change="updateUserRole(user, '{{user.role}}')"></select>
+            <select ng-show="user.editMode" ng-model="user.role" ng-options="role as usersRole.keyFromRole(role) | translate for role in usersRole.availableRoles"></select>
             <span ng-show="user.isUpdatingRole"><i class="fa fa-spinner fa-pulse fa-fw"></i></span>
+            <span ng-click="user.editMode = true" ng-hide="user.editMode">{{ usersRole.keyFromRole(user.role) | translate }}</span>
```

Comment:
Should only the pencil be clickable? Since you have an action icon it might be confusing. I'll ask on JIRA.

Reply:
This functionality was modeled after the email which responded both to a click and an action icon.

Reply:
All good, see comment from Tristan on JIRA